client_admin/pulp/client/admin/tasks.py

```diff
@@ -162,6 +175,33 @@ def cancel(self, **kwargs):
             else:
                 raise
 
+    def purge(self, **kwargs):
+        """
+        Attempts to purge completed tasks with states finished, error, timed out
```

Comment:
Please remove all references to 'timed out' in this PR, as this PR https://github.com/pulp/pulp/pull/1990 will not handle all cleaning once both will get merged.